Makefile

```diff
@@ -29,7 +36,8 @@ $(IO_DIR)/%.sqlite: $(ENV_FILE)
 
 .PHONY: run-api
 run-api: $(IO_DIR)/mdb.sqlite $(IO_DIR)/sdb.sqlite
-	docker run --rm -p 8080:8080 -v$(IO_DIR):/io --env-file $(ENV_FILE) $(IMAGE) --ui --metadata-db=sqlite:///io/mdb.sqlite --state-db=sqlite:///io/sdb.sqlite
+	docker run $(DOCKER_RUN_EXTRA_ARGS) --rm -p 8080:8080 -v$(IO_DIR):/io --env-file $(ENV_FILE) $(IMAGE) --ui --metadata-db=sqlite:///io/mdb.sqlite --state-db=sqlite:///io/sdb.sqlite
```

Comment:
`run-api` was supposed to execute for exclusively development debugging. Any deployment requires a separate config because the logic has many shortcuts for development. Why do we need this?

Reply:
Yup, this is for development only. I used this for debugging after putting some `pdb.set_trace()` in the code. Even if it required to rebuild the docker image, the time for rebuilding is really small.